<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">

<html>
<head>

<title>RGD: Strain Registration Form Help</title>

<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">

<link rel=stylesheet type="text/css" href="/common/style/rgd_styles.css"> 

</head>

<body>

<p><a name="authorLName" id="authorLName"></a><strong>Last name of the PI:</strong></p>
<p>Please enter the last name of the Princiipal investigator of the group.</p>
<p>&nbsp;</p>

<p><a name="authorLFame" id="authorLFame"></a><strong>First name of the PI:</strong></p>
<p>Please enter the first name of the Princiipal investigator of the group.</p>
<p>&nbsp;</p>

<p><a name="authorMFame" id="authorMFame"></a><strong>Initial Middle of the PI: </strong></p>
<p>Please enter the middle name of the Princiipal investigator of the group.</p>
<p>&nbsp;</p>

<p><a name="authorMail" id="authorMail"></a><strong>E-mail address of the PI:</strong></p>
<p>Please enter the e-mail address of the Princiipal investigator of the group </p>
<p>&nbsp;</p>

<p><a name="org" id="org"></a><strong>Institute/Organization:</strong></p>
<p>Name of the institute or organization to which the corresponding author is affiliated</p>
<p>&nbsp;</p>

<p><a name="address" id="address"></a><strong>Address: </strong></p>
<p>The postal address of the corresponding author</p>
<p>&nbsp;</p>

<p><a name="phone" id="phone"></a><strong>Telephone number:</strong></p>
<p>The telephone number of the corresponding author along with the area code.</p>
<p>&nbsp;</p>

<p><a name="fax" id="fax"></a><strong>Fax number:</strong></p>
<p>The fax number of the corresponding author along with the area code.</p>
<p>&nbsp;</p>

<p><a name="userLName" id="userLName"></a><strong>Last name of the submitter:</strong></p>
<p>Please enter the last name of the person submitting the strain data.</p>
<p>&nbsp;</p>

<p><a name="userFName" id="userFName"></a><strong>First name of the submitter:</strong></p>
<p>Please enter the first name of the person submitting the strain data.</p>
<p>&nbsp;</p>

<p><a name="userMName" id="userMName"></a><strong>Middle name of the submitter: </strong></p>
<p>Please enter the middle name of the person submitting the strain data.</p>
<p>&nbsp;</p>

<p><a name="userMail" id="userMail"></a><strong>E-mail address of the submitter: </strong></p>
<p>Please enter the e-mail address of the person submitting the strain data.</p>
<p>&nbsp;</p>

<p><a name="strain_type" id="strain_type"></a><strong>STRAIN TYPE:</strong></p>
<p>This describes the category of the strain i.e. Inbred, Related Inbred, Recombinant Inbred, Coisogenic, Congenic, Consomic, 
Segregating inbred, Conplastic, outbred, mutant, transgenic.  Please use only these terms as this column has a fixed vocabulary. If the type is not mentioned 
please select other and enter the type.</p>
<p>&nbsp;</p>

<p><a name="strain_code" id="strain_code"></a><strong>ILAR LABORATORY CODE:</strong></p>
<p>This is the laboratory code assigned by <a href="http://dels.nas.edu/ilar/codes.asp?id=codes">The Institute of Laboratory and Animal Research</a>
to each lab or group. Registering a group at ILAR identifies the group as a place where this strain was originated and maintained. </p>
<p>&nbsp;</p>

<p><a name="strain" id="strain"></a><strong>STRAIN:</strong></p>
<p>The strain that is being curated is entered here. For example: BN/Cr, SHR.BN-(D13Arb5-Ren)/Ipcv</p>
<p>&nbsp;</p>

<p><a name="sub_strain" id="sub_strain"></a><strong>SUBSTRAIN:</strong></p>
<p>This is the name of the substrain that is derived from the inbred strain (a strain can have many substrains which are genetically 
different from each other). ILAR codes can be used to identify substrains. For example: Cr, Mcwi</p>
<p>&nbsp;</p>

<p><a name="full_name" id="full_name"></a><strong>FULL NAME: </strong> </p>
<p>This is the full name of the strain as used by the author; in some cases it could be the symbol. For example: ACI has the full name ACI.</p>
<p>&nbsp;</p>

<p><a name="donor_strain" id="donor_strain"></a><strong>DONOR STRAIN: </strong> </p>
<p>This is the strain in which the allele, segment or mutation is originated.</p>
<p>&nbsp;</p>

<p><a name="accept_strain" id="accept_strain"></a><strong>ACCEPTOR STRAIN: </strong> </p>
<p>This is the strain which accepts the allele or segment.</p>
<p>&nbsp;</p>

<p><a name="genetic_marker" id="genetic_marker"></a><strong>GENETIC MARKERS: </strong> </p>
<p>This column holds information related to any genetic marker/gene, which is used to generate the strain. The symbols of the commonly used markers are: </p>
<table>
<tr><td><strong>Symbol<strong></td><td><strong>&nbsp;&nbsp;&nbsp;&nbsp;Name</strong></td></tr>
<tr><td>A</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Agouti</td></tr>
<tr><td>a</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Non-agouti</td></tr>
<tr><td>H</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Non-hooded</td></tr>
<tr><td>h</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Hooded</td></tr>
<tr><td>hi</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Irish hooding (much more black than h)</td></tr>
<tr><td>P</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Non-pink-eyed (ie, wild type)</td></tr>
<tr><td>p</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Pink-eyed dilute</td></tr>
</table>

<p><a name="strain_color" id="strain_color"></a><strong>STRAIN COLOR:</strong> </p>
<p>This column holds information about the coat color. The examples of the symbols used are:</p>
<table>
<tr><td><strong>Symbol<strong></td><td><strong>&nbsp;&nbsp;&nbsp;&nbsp;Name</strong></td></tr>
<tr><td>a</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Agouti</td></tr>
<tr><td>b</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;brown</td></tr>
<tr><td>c</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;albino</td></tr>
<tr><td>d</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;dilute</td></tr>
<tr><td>e</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;yellow</td></tr>
<tr><td>f</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;fawn</td></tr>
<tr><td>h</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;hooded</td></tr>
<tr><td>Mi</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;white</td></tr>
<tr><td>Mib</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;microphthalmie-blanc</td></tr>
<tr><td>p</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;Pink-eyed yellow</td></tr>
<tr><td>r</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;red-eyed yellow</td></tr>
<tr><td>s</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;silver</td></tr>
<tr><td>sd</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;sand</td></tr>
<tr><td>sl</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;spotted lethal</td></tr>
<tr><td>wb</td> <td>&nbsp;&nbsp;&nbsp;&nbsp;white belly</td></tr>
</table>
					
<p><a name="inbred_gen" id="inbred_gen"></a><strong>INBRED_GEN:</strong> </p>
<p>This is the number of generations this strain has been inbred (the last numeric count). For example F32.</p>
<p>&nbsp;</p>

<p><a name="strain_origin" id="strain_origin"></a><strong>STRAIN ORIGIN:</strong> </p>
<p>Specific information regarding the origin of this strain is entered in this column. This field could have information like the 
institution, lab, or company where this strain was originated. For example: 'BN/NHsdMcwi is a Brown Norway strain which came from 
Ss (now a retired ILAR code) to N (NIH) then to Hsd (Harlan Sprague Dawley) and now it is in Mcwi (Medical College of Wisconsin)'.</p>
<p>&nbsp;</p>

<p><a name="reproduction" id="reproduction"></a><strong>REPRODUCTION:</strong> </p>
<p>Specifications related to reproductive characteristics of the strain; may include information like the litter size. For example: 
Poor reproductive performance, homozygous female have normal fertility.</p>
<p>&nbsp;</p>

<p><a name="general_char" id="general_char"></a><strong>GENERAL CHARACTERISTICS:</strong> </p>
<p>Individual characteristics associated to the particular strain being curated are entered here. (Any information, which could not be entered in any of the categories, mentioned below). For example: spots of color pigmentation observed on the head and tail.</p>
<p>&nbsp;</p>

<p><a name="behavior" id="behavior"></a><strong>BEHAVIOR:</strong> </p>
<p>Notes related to the specific behavior of the strain. For example: hyperactive, avoidance in learning.</p>
<p>&nbsp;</p>

<p><a name="lifespan" id="lifespan"></a><strong>LIFESPAN AND SPONTANEOUS DISEASE:</strong> </p>
<p>Any information regarding the life span and diseases that occur in the strain of interest. For example: An age-related glomerulosclerosis occurs in this strain.</p>
<p>&nbsp;</p>

<p><a name="anatomy" id="anatomy"></a><strong>ANATOMY:</strong> </p>
<p>Anatomy of the strain. For example: Does not develop brownish skin scales on the dorsum of the body, the perineum and the tail, and low relative heart weight in 10-week old males.</p>
<p>&nbsp;</p>

<p><a name="infection" id="infection"></a><strong>INFECTION:</strong> </p>
<p>Infections that can occur in the strain. For example: Infection with Hymenolepsis diminuata cysticercoids results in no worm loss and no mastocytosis.</p>
<p>&nbsp;</p>

<p><a name="immunology" id="immunology"></a><strong>IMMUNOLOGY:</strong> </p>
<p>Immunity of drugs and chemicals to the strain. For example: Low antibody response to streptococcal groups a carbohydrate, not MHC linked.</p>
<p>&nbsp;</p>

<p><a name="physiology" id="physiology"></a><strong>PHYSIOLOGY AND BIOCHEMISTRY:</strong> </p>
<p>This column holds information regarding physiological and biochemical characteristics of the strain. For example: Low serum growth hormone, Low blood pressure, Resistant to adrenal regeneration hypertension.</p>
<p>&nbsp;</p>

<p><a name="drug" id="drug"></a><strong>DRUGS AND CHEMICALS: </strong> </p>
<p>This holds information related to the interaction and response of drugs and chemicals on the strain. For example: Dimethylbenzanthracene induced a transplantable myeloid leukaemia, and intermediate susceptibility to pentobarbital sodium (3/7) with LD50 of 90mg/kg.</p>
<p>&nbsp;</p>

<p><a name="altered" id="altered"></a><strong>CHR_ALTERED:</strong> </p>
<p>If a chromosome is altered during the generation of the strain being curated, the chromosome number has to be entered here. Should be definitely completed while entering a congenics/consomic strain.</p>
<p>&nbsp;</p>

<p><a name="marker1" id="marker1"></a><strong>FLANK_MARKER_1:</strong> </p>
<p>The RGD_ID of the marker used during the generation of a congenic strain.</p>
<p>&nbsp;</p>

<p><a name="marker2" id="marker2"></a><strong>FLANK_MARKER_2:</strong> </p>
<p>The RGD_ID of the marker used during the generation of a congenic strain.</p>
<p>&nbsp;</p>

<p><a name="gene_rgd_id" id="gene_rgd_id"></a><strong>GENE_RGD_ID:</strong> </p>
<p>If the strain is related to any gene in RGD please enter the RGD_ID of that gene this is required for making the link between the strain and the related gene.</p>
<p>&nbsp;</p>

<p><a name="gene_symbol" id="gene_symbol"></a><strong>GENE_SYMBOL:</strong> </p>
<p>The symbol of the gene, which is related to this strain.</p>
<p>&nbsp;</p>

<p><a name="sslp_rgd_id" id="sslp_rgd_id"></a><strong>SSLP_RGD_ID:</strong> </p>
<p>If any SSLP is associated to this strain please enter the RGD_ID of the SSLP.</p>
<p>&nbsp;</p>

<p><a name="sslp_symbol" id="sslp_symbol"></a><strong>SSLP_SYMBOL:</strong> </p>
<p>Symbol of the SSLP associated to the strain is entered here.</p>
<p>&nbsp;</p>

<p><a name="qtl_rgd_id" id="qtl_rgd_id"></a><strong>QTL_RGD_ID: </strong> </p>
<p>If any QTL is associated to the strain then enter the RGD_ID of the QTL here.</p>
<p>&nbsp;</p>

<p><a name="qtl_symbol" id="qtl_symbol"></a><strong>QTL_SYMBOL: </strong> </p>
<p>Symbol of the QTL associated to the strain is entered here.</p>
<p>&nbsp;</p>

<p><a name="note" id="note"></a><strong>NOTES: </strong> </p>
<p>Any other information related to the strain. This holds any other relevant information that is of help to the curator and is not displayed on the report page.</p>
<p>&nbsp;</p>

<p><a name="source" id="source"></a><strong>SOURCE: </strong> </p>
<p>The name of the lab, department, University and even the company from where this particular strain can be obtained. Please give the complete details including the address if possible. Please enter the URL of the source.</p>
<p>&nbsp;</p>

</body>

</html>

